app/models/filter.rb

```diff
@@ -165,7 +165,22 @@ def enforce_inherited_taxonomies
   def inherit_taxonomies!
     self.organization_ids = role.organization_ids if resource_taxable_by_organization?
     self.location_ids = role.location_ids if resource_taxable_by_location?
-    build_taxonomy_search
+    build_taxonomy_search_from_ids
```

Comment:
Would keeping this and changing L198 in the new diff to use map instead of pluck help?

Reply:
Changing just `build_taxonomy_search` seem to work as well for create and edit filter